### PR TITLE
Test for azure-ts-appservice and test fixes

### PR DESCRIPTION
**Pull request by daveremy**

### CODE REVIEW

File: azure-ts-appservice/README.md [hunk 1]
```diff
@@ -28,10 +28,9 @@ with App Service.
 1. Define SQL Server password (make it complex enough to satisfy Azure policy):
 
     ```
-    pulumi config set <stack>:sqlPassword <value>
+    pulumi config set --secret <stack>:sqlPassword <value>
```
Comment (lukehoban) [inline azure-ts-appservice/README.md]: Note that we should be able to remove the `<stack>` part here as well.

File: azure-ts-appservice/README.md [hunk 2]
```diff
@@ -50,7 +49,11 @@ with App Service.
 
     ```
     $ pulumi stack output endpoint
-    https://azpulumi-as0ef47193.azurewebsites.net
+    https://azpulumi-as0ef47193.azurewebsites.net (you will get a different url)
```
Comment (lukehoban) [inline azure-ts-appservice/README.md]: Don't think we have to add this here (we have similar many other places as well).

File: azure-ts-appservice/README.md [hunk 3]
```diff
@@ -50,7 +49,11 @@ with App Service.
...
     $ curl "$(pulumi stack output endpoint)"
-    Greetings from Azure App Service!
+    <html>
+        <body>
+            <h1>Greetings from Azure App Service!</h1>
+        </body>
+    </html>%
```
Comment (lukehoban) [inline azure-ts-appservice/README.md]: What's the `%` for?

File: azure-ts-appservice/README.md (hunk 1 above)
Comment (daveremy) [reply azure-ts-appservice/README.md]: it needs to be `pulumi config set --secret azure-appservice:sqlPassword <value>` right?

File: azure-ts-appservice/README.md (hunk 3 above)
Comment (daveremy) [reply azure-ts-appservice/README.md]: cut and paste issue probably

File: azure-ts-appservice/README.md (hunk 1 above)
Comment (lukehoban) [reply azure-ts-appservice/README.md]: You can leave that off and it will default to the project-scoped config.

**Pull request merged**